### PR TITLE
[Bulk] Validation: Fix invalid ms.value

### PR DESCRIPTION
**Pull request by reemsabaw1**
This PR fixes 70 issues in microsoft-graph-docs.

### CODE REVIEW

**Comment by reemsabaw1** [comment]
@Lauragra 



The Content & Learning team is fixing build validation errors on docs.microsoft.com for the rest of H2. This will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. Please review within five business days and merge or comment in the PR with any changes you’d like to see. Thank you!

File: api-reference/beta/resources/root.md [hunk 1]
```diff
@@ -5,7 +5,8 @@ ms.date: 09/10/2017
 title: Root
 ms.localizationpriority: medium
 doc_type: resourcePageType
-ms.prod: ""
+ms.prod: cloud-pc
+ms.technology: microsoft-graph
```
Comment (Lauragra) [inline api-reference/beta/resources/root.md:9]: Please globally remove this technology value; it is set globally.

**Comment by reemsabaw1** [comment]
#feedback-addressed

**Comment by v-emilypr** [comment]
Hi @Lauragra, are there any remaining changes you would like to see on this PR before we can merge? Thanks for all your reviews!

File: api-reference/beta/resources/root.md [hunk 2]
```diff
@@ -5,7 +5,7 @@ ms.date: 09/10/2017
 title: Root
 ms.localizationpriority: medium
 doc_type: resourcePageType
-ms.prod: ""
+ms.prod: cloud-pc
```
Comment (Lauragra) [inline api-reference/beta/resources/root.md:8]: This value should be sites-and-lists.

File: api-reference/beta/resources/searchresult.md [hunk 3]
```diff
@@ -5,7 +5,7 @@ ms.date: 09/10/2017
 title: SearchResult
 ms.localizationpriority: medium
 doc_type: resourcePageType
-ms.prod: ""
+ms.prod: cloud-pc
```
Comment (Lauragra) [inline api-reference/beta/resources/searchresult.md:8]: files

File: api-reference/beta/resources/sectionlinks.md [hunk 4]
```diff
@@ -3,7 +3,7 @@ title: "sectionLinks resource type"
 description: "Links for opening a OneNote section."
 ms.localizationpriority: medium
 doc_type: resourcePageType
-ms.prod: ""
+ms.prod: cloud-pc
```
Comment (Lauragra) [inline api-reference/beta/resources/sectionlinks.md:6]: notes

File: api-reference/beta/resources/securescorecontrolprofiles.md [hunk 5]
```diff
@@ -3,7 +3,8 @@ title: "secureScoreControlProfile resource type"
 description: "Represents a tenant's secure score per control data. By default, it returns all controls for a tenant and can explicitly pull individual controls."
 ms.localizationpriority: medium
 doc_type: resourcePageType
-ms.prod: security
+
+ms.prod: cloud-pc
```
Comment (Lauragra) [inline api-reference/beta/resources/securescorecontrolprofiles.md:7]: This value should be security, not sure why the change.

File: api-reference/beta/resources/securescorecontrolstateupdate.md [hunk 6]
```diff
@@ -3,7 +3,7 @@ title: " secureScoreControlStateUpdate resource type"
 description: "This resource contains history of control states updated by user (control states include Default, Ignored, ThirdParty, Reviewed)."
 ms.localizationpriority: medium
 doc_type: resourcePageType
-ms.prod: security
+ms.prod: cloud-pc
```
Comment (Lauragra) [inline api-reference/beta/resources/securescorecontrolstateupdate.md:6]: security

File: api-reference/beta/resources/securescores.md [hunk 7]
```diff
@@ -3,7 +3,7 @@ title: "secureScore resource type"
 description: "top=n, where n = the number of days of data that you want to retrieve. "
 ms.localizationpriority: medium
 doc_type: resourcePageType
-ms.prod: security
+ms.prod: cloud-pc
```
Comment (Lauragra) [inline api-reference/beta/resources/securescores.md:6]: security

File: api-reference/beta/resources/shareaction.md [hunk 8]
```diff
@@ -5,7 +5,7 @@ ms.date: 09/14/2017
 title: ShareAction
 ms.localizationpriority: medium
 doc_type: resourcePageType
-ms.prod: ''
+ms.prod: cloud-pc
```
Comment (Lauragra) [inline api-reference/beta/resources/shareaction.md:8]: files

File: api-reference/beta/resources/shared.md [hunk 9]
```diff
@@ -1,11 +1,11 @@
...
+title: Shared
 author: JeremyKelley
-description: "The Shared resource indicates a DriveItem has been shared with others."
+description: The Shared resource indicates a DriveItem has been shared with others.
 ms.date: 09/10/2017
-title: Shared
 ms.localizationpriority: medium
 doc_type: resourcePageType
-ms.prod: ""
+ms.prod: cloud-pc
```
Comment (Lauragra) [inline api-reference/beta/resources/shared.md:8]: files

File: api-reference/beta/resources/shareddriveitem.md [hunk 10]
```diff
@@ -5,7 +5,7 @@ ms.date: 09/10/2017
 title: SharedDriveItem
 ms.localizationpriority: medium
 doc_type: resourcePageType
-ms.prod: ''
+ms.prod: cloud-pc
```
Comment (Lauragra) [inline api-reference/beta/resources/shareddriveitem.md:8]: files

File: api-reference/beta/resources/sharinginvitation.md [hunk 11]
```diff
@@ -5,7 +5,7 @@ ms.date: 09/10/2017
 title: SharingInvitation
 ms.localizationpriority: medium
 doc_type: resourcePageType
-ms.prod: ''
+ms.prod: cloud-pc
```
Comment (Lauragra) [inline api-reference/beta/resources/sharinginvitation.md:8]: files

File: api-reference/beta/resources/sharinglink.md [hunk 12]
```diff
@@ -5,7 +5,7 @@ ms.date: 09/10/2017
 title: SharingLink
 ms.localizationpriority: medium
 doc_type: resourcePageType
-ms.prod: ""
+ms.prod: cloud-pc
```
Comment (Lauragra) [inline api-reference/beta/resources/sharinglink.md:8]: files

File: api-reference/beta/resources/sizerange.md [hunk 13]
```diff
@@ -3,7 +3,7 @@ title: "sizeRange resource type"
 description: "Specifies the maximum and minimum sizes (in kilobytes) that an incoming message must have in order for a condition or exception to apply."
 ms.localizationpriority: medium
 doc_type: resourcePageType
-ms.prod: ""
+ms.prod: cloud-pc
```
Comment (Lauragra) [inline api-reference/beta/resources/sizerange.md:6]: mail

File: api-reference/beta/resources/standardtimezoneoffset.md [hunk 14]
```diff
@@ -3,7 +3,7 @@ title: "standardTimeZoneOffset resource type"
 description: "Specifies when a time zone switches from daylight saving time to standard time."
 ms.localizationpriority: medium
 doc_type: resourcePageType
-ms.prod: ""
+ms.prod: cloud-pc
```
Comment (Lauragra) [inline api-reference/beta/resources/standardtimezoneoffset.md:6]: mail

File: api-reference/beta/resources/storageplaninformation.md [hunk 15]
```diff
@@ -5,7 +5,7 @@ ms.date: 06/20/2018
 title: StoragePlanInformation
 ms.localizationpriority: medium
 doc_type: resourcePageType
-ms.prod: ''
+ms.prod: cloud-pc
```
Comment (Lauragra) [inline api-reference/beta/resources/storageplaninformation.md:8]: files

File: api-reference/beta/resources/systemfacet.md [hunk 16]
```diff
@@ -1,21 +1,22 @@
 ---
 author: daspek
-description: The **system** facet indicates that the object is managed by the system for its own operation.
+description: <descripton>
+ms.date: 09/12/2017
 title: SystemFacet
 ms.localizationpriority: medium
 doc_type: resourcePageType
-ms.prod: sites-and-lists
+ms.prod: cloud-pc
```
Comment (Lauragra) [inline api-reference/beta/resources/systemfacet.md:8]: sites-and-lists

File: api-reference/beta/resources/targetresourceapp.md [hunk 17]
```diff
@@ -3,7 +3,7 @@ title: "targetResourceApp resource type"
 description: "Indicates the app object that was added, updated or deleted by admins as part of audit activity. Derived from the targetResource resource."
 ms.localizationpriority: medium
 doc_type: resourcePageType
-ms.prod: ""
+ms.prod: cloud-pc
```
Comment (Lauragra) [inline api-reference/beta/resources/targetresourceapp.md:6]: identity-and-access-reports

File: api-reference/beta/resources/tasks-identifiers-disclaimer.md [hunk 18]
```diff
@@ -3,7 +3,7 @@ title: "Identifiers in Tasks"
 description: "Identifiers for objects in Tasks are service-generated string values. . The values are 28 characters long and are case-sensitive. When passed as in, the service will do a simple format validation of the identifier, if the format validation fails, the callers will receive a Bad Request (400) error response indicating this problem. Receiving this error indicates a bug in the calling application, such as:"
 ms.localizationpriority: medium
 doc_type: conceptualPageType
-ms.prod: ""
+ms.prod: cloud-pc
```
Comment (Lauragra) [inline api-reference/beta/resources/tasks-identifiers-disclaimer.md:6]: tasks-and-plans

File: api-reference/beta/resources/termstore-group.md [hunk 19]
```diff
@@ -4,7 +4,7 @@ title: Group resource type
 doc_type: "resourcePageType"
 description: "Represents a group used in a term store."
 ms.localizationpriority: medium
-ms.prod: "taxonomy"
+ms.prod: cloud-pc
```
Comment (Lauragra) [inline api-reference/beta/resources/termstore-group.md:7]: taxonomy

File: api-reference/beta/resources/termstore-localizeddescription.md [hunk 20]
```diff
@@ -3,7 +3,7 @@ title: "localizedDescription resource type"
 description: "Represents the localized description used to describe a term in the term store."
 author: mohitpcad
 ms.localizationpriority: medium
-ms.prod: "Sharepoint"
+ms.prod: sharepoint
```
Comment (Lauragra) [inline api-reference/beta/resources/termstore-localizeddescription.md:6]: sites-and-lists

File: api-reference/beta/resources/termstore-localizedlabel.md [hunk 21]
```diff
@@ -3,7 +3,7 @@ title: "localizedLabel resource type"
 description: "Represents the label for a term in the term store."
 author: mohitpcad
 ms.localizationpriority: medium
-ms.prod: "Sharepoint"
+ms.prod: sharepoint
```
Comment (Lauragra) [inline api-reference/beta/resources/termstore-localizedlabel.md:6]: sites-and-lists

File: api-reference/beta/resources/termstore-localizedname.md [hunk 22]
```diff
@@ -3,7 +3,7 @@ title: "localizedName resource type"
 description: "Represents the localized name used in the term store, which identifies the name in the localized language."
 author: mohitpcad
 ms.localizationpriority: medium
-ms.prod: "sharepoint-taxonomy"
+ms.prod: sharepoint
```
Comment (Lauragra) [inline api-reference/beta/resources/termstore-localizedname.md:6]: sites-and-lists

File: api-reference/beta/resources/termstore-set.md [hunk 23]
```diff
@@ -3,7 +3,7 @@ title: "set resource type"
 description: "Represents a set in a term store."
 author: mohitpcad
 ms.localizationpriority: medium
-ms.prod: "Sharepoint"
+ms.prod: sharepoint
```
Comment (Lauragra) [inline api-reference/beta/resources/termstore-set.md:6]: sites-and-lists

File: api-reference/beta/resources/termstore-store.md [hunk 24]
```diff
@@ -3,7 +3,7 @@ title: "store resource type"
 description:  "Represents a taxonomy term store."
 author: mohitpcad
 ms.localizationpriority: medium
-ms.prod: "Sharepoint"
+ms.prod: sharepoint
```
Comment (Lauragra) [inline api-reference/beta/resources/termstore-store.md:6]: sites-and-lists

File: api-reference/beta/resources/termstore-term.md [hunk 25]
```diff
@@ -3,7 +3,7 @@ title: "term resource type"
 description: "Defines a term entity in a term store."
 author: mohitpcad
 ms.localizationpriority: medium
-ms.prod: "Sharepoint"
+ms.prod: sharepoint
```
Comment (Lauragra) [inline api-reference/beta/resources/termstore-term.md:6]: sites-and-lists

File: api-reference/beta/resources/textColumn.md [hunk 26]
```diff
@@ -5,7 +5,7 @@ ms.date: 09/11/2017
 title: TextColumn
 ms.localizationpriority: medium
 doc_type: resourcePageType
-ms.prod: ''
+ms.prod: cloud-pc
```
Comment (Lauragra) [inline api-reference/beta/resources/textColumn.md:8]: workbooks-and-charts

File: api-reference/beta/resources/thumbnail.md [hunk 27]
```diff
@@ -5,7 +5,7 @@ ms.date: 09/10/2017
 title: Thumbnail
 ms.localizationpriority: medium
 doc_type: resourcePageType
-ms.prod: ""
+ms.prod: cloud-pc
```
Comment (Lauragra) [inline api-reference/beta/resources/thumbnail.md:8]: sites-and-lists

File: api-reference/beta/resources/thumbnailset.md [hunk 28]
```diff
@@ -5,7 +5,7 @@ ms.date: 09/10/2017
 title: ThumbnailSet
 ms.localizationpriority: medium
 doc_type: resourcePageType
-ms.prod: ""
+ms.prod: cloud-pc
```
Comment (Lauragra) [inline api-reference/beta/resources/thumbnailset.md:8]: sites-and-lists

File: api-reference/beta/resources/timestamp.md [hunk 29]
```diff
@@ -3,7 +3,7 @@ title: "timeStamp resource type"
 description: "Date and time information for a point in time."
 ms.localizationpriority: medium
 doc_type: resourcePageType
-ms.prod: ""
+ms.prod: cloud-pc
```
Comment (Lauragra) [inline api-reference/beta/resources/timestamp.md:6]: sites-and-lists

File: api-reference/beta/resources/timezonebase.md [hunk 30]
```diff
@@ -3,7 +3,7 @@ title: "timeZoneBase resource type"
 description: "The basic representation of a time zone."
 ms.localizationpriority: medium
 doc_type: resourcePageType
-ms.prod: ""
+ms.prod: cloud-pc
```
Comment (Lauragra) [inline api-reference/beta/resources/timezonebase.md:6]: mail

File: api-reference/beta/resources/timezoneinformation.md [hunk 31]
```diff
@@ -3,7 +3,7 @@ title: "timeZoneInformation resource type"
 description: "Represents a time zone. The supported format is Windows, and Internet Assigned Numbers Authority (IANA) time zone (also known as Olson time zone)"
 ms.localizationpriority: medium
 doc_type: resourcePageType
-ms.prod: ""
+ms.prod: cloud-pc
```
Comment (Lauragra) [inline api-reference/beta/resources/timezoneinformation.md:6]: mail

File: api-reference/beta/resources/typedemailaddress.md [hunk 32]
```diff
@@ -3,7 +3,7 @@ title: "typedEmailAddress resource type"
 description: "Represents the name, email addresses, and their corresponding email address type of a contact."
 ms.localizationpriority: medium
 doc_type: resourcePageType
-ms.prod: ""
+ms.prod: cloud-pc
```
Comment (Lauragra) [inline api-reference/beta/resources/typedemailaddress.md:6]: mail

File: api-reference/beta/resources/versionaction.md [hunk 33]
```diff
@@ -5,7 +5,7 @@ ms.date: 09/14/2017
 title: VersionAction
 ms.localizationpriority: medium
 doc_type: resourcePageType
-ms.prod: ''
+ms.prod: cloud-pc
```
Comment (Lauragra) [inline api-reference/beta/resources/versionaction.md:8]: files

File: api-reference/beta/resources/video.md [hunk 34]
```diff
@@ -5,7 +5,7 @@ ms.date: 09/10/2017
 title: Video
 ms.localizationpriority: medium
 doc_type: resourcePageType
-ms.prod: ''
+ms.prod: cloud-pc
```
Comment (Lauragra) [inline api-reference/beta/resources/video.md:8]: files

File: api-reference/beta/resources/vulnerabilitystate.md [hunk 35]
```diff
@@ -3,7 +3,7 @@ title: "vulnerabilityState resource type"
 description: " > **Important:** APIs under the /beta version in Microsoft Graph are in preview and are subject to change. Use of these APIs in production applications is not supported."
 ms.localizationpriority: medium
 doc_type: resourcePageType
-ms.prod: ""
+ms.prod: cloud-pc
```
Comment (Lauragra) [inline api-reference/beta/resources/vulnerabilitystate.md:6]: security

File: api-reference/beta/resources/website.md [hunk 36]
```diff
@@ -3,7 +3,7 @@ title: "website resource type"
 description: "Represents a web site."
 ms.localizationpriority: medium
 doc_type: resourcePageType
-ms.prod: ""
+ms.prod: cloud-pc
```
Comment (Lauragra) [inline api-reference/beta/resources/website.md:6]: mail

File: api-reference/beta/resources/workposition.md [hunk 37]
```diff
@@ -3,7 +3,7 @@ title: "workPosition resource type"
 description: "workPosition resource type"
 ms.localizationpriority: medium
 author: "kevinbellinger"
-ms.prod: "people"
+ms.prod: people
```
Comment (Lauragra) [inline api-reference/beta/resources/workposition.md:6]: people-and-workplace-intelligence

**Comment by Lauragra** [comment]
> Hi @Lauragra, are there any remaining changes you would like to see on this PR before we can merge? Thanks for all your reviews!

@v-emilypr , FYI, I added comments indicating the correct values to use. Thanks.

**Comment by reemsabaw1** [comment]
> > Hi @Lauragra, are there any remaining changes you would like to see on this PR before we can merge? Thanks for all your reviews!
> 
> @v-emilypr , FYI, I added comments indicating the correct values to use. Thanks.

Thank you for the review! I have addressed the changes requested.

**Comment by jmart1428** [comment]
Hi, @Lauragra, just checking in to see if there are any other changes we need to make on this PR? Thank you for your help!

**Review by Lauragra — APPROVED** [review]
